packages/odyssey-design-tokens/src/font.json

```diff
@@ -10,15 +10,31 @@
         "value": "'Inconsolata', 'SFMono-Regular', 'Consolas', 'Liberation Mono', 'Menlo', 'Courier', monospace"
       }
     },
+    "scale": {
+      "0": { "value": "0.857rem" },
+      "1": { "value": "1rem" },
+      "2": { "value": "1.143rem" },
+      "3": { "value": "1.286rem" },
+      "4": { "value": "1.429rem" },
+      "5": { "value": "1.571rem" },
+      "6": { "value": "1.786rem" },
+      "7": { "value": "2rem" },
+      "8": { "value": "2.286rem" },
+      "9": { "value": "2.571rem" },
+      "10": { "value": "2.857rem" },
+      "11": { "value": "3.214rem" },
+      "12": { "value": "3.643rem" }
```

Comment:
Similar to color, Design expects access to the full "palette" for cases where a functional token isn't available. For ex: we don't use Scales 9+ in the system yet, but designers want an escape hatch for huge type.

Reply:
Its ok if design wants access to all of these via figma, but should develops have access to them in code is my question? I'm ok with opening everything up for what it is worth but I know traditionally some of these tokens were never intended to be used which we should absolutely prevent using scripting.

Reply:
I think the intent is for devs to have them as well. If a designer hands off a file using FontScale12, this would currently be the only way for a dev to grab that value (without adding it via bespoke code themselves).